src/components/MonthsSlider.tsx

```diff
@@ -58,7 +58,7 @@ export default function MonthsSlider({
       setMonthRange(range);
       setDates({ min, max });
     },
-    [],
+    [globalDates.min, setDates],
```

Comment:
how is setDates here ? it's a setter not a variable.

Reply:
Setters, as all functions in javascript, are values and their pointers can change. This is why we use `useCallback`, for instance. If this component were to receive a different setter, it would need to change its callback.

also, this was raising a warning from the linter, and I believe we should address linter warnings.

Reply:
setMonthRange is stable by definition. The linter knows this since [this PR from 2020](https://github.com/facebook/react/pull/19590). This component cannot know about the stability of its props.